File: 455-assign-cookies/assign-cookies.py
class Solution:
    def findContentChildren(self, g: List[int], s: List[int]) -> int:
        n = len(g)
        m = len(s)
        count = 0
        g.sort()
        s.sort()

        i, j = 0, 0

        while(i < n and j < m):
            if(g[i] <= s[j]):
                count += 1
                i += 1
            j += 1
                
        return count

File: 455-assign-cookies/assign-cookies.js
/**
 * @param {number[]} g
 * @param {number[]} s
 * @return {number}
 */
var findContentChildren = function(g, s)
{
    g.sort((a, b) => a - b);
    s.sort((a, b) => a - b);
    n = g.length
    m = s.length

    let i = 0, j = 0

    while(i < n && j < m)
    {
        if(g[i] <= s[j])
        {
            i += 1
        }
        j += 1
    }

    return i
};